SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -27,10 +27,174 @@
 #include "Modules/ModuleManager.h"
 #include "Net/UnrealNetwork.h"
 
+#if UE_EDITOR
+#include "Editor.h"
+#endif
+
 using namespace SpatialGDK;
 
 DEFINE_LOG_CATEGORY(LogSpatialDebugger);
 
+FSpatialDebuggerSystem::FSpatialDebuggerSystem(USpatialNetDriver* InNetDriver, const FSubView& InSubView)
+	: NetDriver(InNetDriver)
+	, SubView(&InSubView)
+{
+	check(IsValid(InNetDriver));
+
+	EntityActorMapping.Reserve(ENTITY_ACTOR_MAP_RESERVATION_COUNT);
```

Comment:
nit: This container is only used on clients, no need to reserve or iterate on servers